### ISSUE TITLE
Volume Slider in User Context Menu Is Not Reported with Screenreaders

### ISSUE THREAD

**Issue by hjozwiak** [mumble-voip/mumble#6109]
### Description

When going through the context menu to adjust the local volume of another user, the slider that governs the volume is not reported.

### Steps to reproduce

1. Open the context menu on a user.
2. Arrow down four times.
3. This will take you to the slider, which is not spoken.
4. Further presses of the up/down arrow keys will increase/decrease the volume.

### Mumble version

Mumble 1.6.0 at commit 9c492fe0

### Mumble component

Client

### OS

Linux

### Reproducible?

Yes

### Additional information

There seems to be a gap between the slider moving and the volume being adjusted; I'm not sure if those are empty elements or something else. You can escape the slider with tab or shift-tab, depending on which direction you want to go in the menu.

**Comment by Hartmnt** [comment]
Hi. Thank you very much for you bug reports. I know that accessibility is currently sub-par in Mumble, but rest assured it is actively being worked on. I have been personally working on it for a while now, on and off. If you would like to take a look, the merge request #5984 contains a hopefully screen reader friendly checklist of a11y improvements. The volume slider is also part of that ToDo list. Not all changes are completed yet, but if you like I can ping you here on GitHub once I have a release candidate. The a11y changes will sadly not make it into the 1.5 stable release of Mumble though.

**Comment by Kissaki** [comment]
@Hartmnt That PR seems like a complex, exhaustive effort. I feel like a [what I assume to be] simple labeling issue like this should be fixed outside of that. It is independent, and would further increase the PR size and complexity, and it could be landed earlier.

_Changeset scope should be kept narrow to drive improvements and development forward. Most of the time it is preferable to do small iterations over big exhaustive changesets - for multiple reasons._

**Comment by Hartmnt** [comment]
> @Hartmnt That PR seems like a complex, exhaustive effort. I feel like a [what I assume to be] simple labeling issue like this should be fixed outside of that. It is independent, and would further increase the PR size and complexity, and it could be landed earlier.
> 
> _Changeset scope should be kept narrow to drive improvements and development forward. Most of the time it is preferable to do small iterations over big exhaustive changesets - for multiple reasons._

Sadly the accessibility options in Qt are rather limited. For the inline volume slider in particular it seems to be almost impossible to get accessible. I have had to hack some event filters to automatically switch to the next option when pressing certain keys, because otherwise it would simply keep and lock the focus on the slider. Besides that I have yet to find a way to make it actually read ANY value when changed.

Regarding the commit advice: In general I would agree. In this certain case I strongly disagree. Mumbles accessibility has been neglected for years it seems, and because of that it is not as simple as changing a few labels and accessibleNames around. If I had to create a separate merge-request for every small and bigger change and then of course first create a dependency list of the additional features and widgets - because I want to share duplicate code instead of doing the same thing multiple times, I would rather not work on this at all to be honest.
If you have any constructive advice for the unchecked problems in the merge request ToDo list, I would highly appreciate the input there, as I feel like this discussion is out of scope of this particular issue here.
